examples/prometheus/README.md

```diff
@@ -0,0 +1,44 @@
+# Prometheus Example
+
+This example demonstrates how to use the OpenTelemetry SDK 
+to instrument a simple application using Prometheus as the metric exporter and expose the metrics via HTTP. 
+
+These are collected by a Prometheus instance which is configured to pull these  metrics via HTTP. 
+
+# How to run
+
+## Prerequisites
+* Java 1.7
+* Docker 19.03
+
+## 1 - Compile 
+```shell script
+../gradlew fatJar
+```
+## 2 - Run Prometheus
+
+Start Prometheus instance with a configuration that sets up a HTTP collection job for  ```127.0.0.1:19090```
+`
+
+See [prometheus.yml](prometheus.yml)
+
+```shell script
+docker run --network="host" --rm -it \
+    --name prometheus \
+    -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus 
```

Comment:
```suggestion
    -p 9090:9090 \
    prom/prometheus 
```

Reply:
I left the published ports out of the run command as these are ignored, as the container is run using ```--network="host"``` mode.

Reply:
Oh, I miss that! You are right! :D